packs/elastic/README.md

```diff
@@ -0,0 +1,170 @@
+# Elasticsearch integration pack
+
+Pack provides many operations helping to manage your Elasticsearch indices. Current functionality is mostly based on [curator](http://www.elastic.co/guide/en/elasticsearch/client/curator/current/).
+
+## Features
+
+Pack provides a set of actions based on curator.
+
+## Curator based actions
+
+Action | Description
+------ | -----------
+**alias** | Add indices to or remove them from aliases.
+**allocation** | Set routing allocation based on tags.
+**bloom** | Disable the bloom filter cache for indices.
+**close** | Close indices.
+**delete.indices** | Delete indices.
+**delete.snapshots** | Delete snapshots.
+**open** | Open indices.
```

Comment:
Yeah, I think 4 levels are too much, but I personally think "curator" shouldn't be included in the name.

This code using curator library is just an implementation detail. User shouldn't and doesn't care how it's implement - they just want to perform an action (e.g. optimize an index, delete a document, etc.). It doesn't matter to them how the action is implemented so having curator in the name, imo, doesn't make much sense and it adds confusion.

On the other hand, having a resource type on which the action operates on in the name does make sense imo - e.g. I want to find all the available actions on a document. If we follow the same naming convention, user can just search for actions starting with `elasticsearch.documents`.